### ISSUE TITLE
Possible accessibility improvements

### ISSUE THREAD

**Issue by renestalder** [youngcut/kirby-form-blocks#8]
This is a great plugin. I'd say, forms consume most of my time to code manually, and this plugin hits the right spot. I will use it for the first time and I will probably make some accessibility improvements:

- **Add formatting (including validation markers) rules as HTML properties**: For example, the fields do not use the `required` HTML attribute even though the required-toggle is activated. Users of assistive technology will not be able to detect that the fields are required. Same goes for a couple of rules like min/max. (in the same term, adding `novalidate` to the form element will ensure we don't use the browser built in validation handlers).
- **Move inline error messages to the bottom of the field and add `aria-invalid="true"` plus `aria-describedby="errorMessageId"` to the failing input field**: This ensures that a screenreader hits the form field and then informs the user that the field is invalid, plus gets can read the associated error description.
- **Move error summary to top of the form**: Ensures that a screenreader user has this summary as the first information.

These are the improvements that can be made I detected so far. And I will probably deliver a pull request for those changes. These will be breaking changes as the source order changes.

**Comment by youngcut** [comment]
Hello again. ;)
Nice to see to have created something that makes life easier for others and thanks for your inputs. 
I would be very happy for a pull request. I'm pretty busy at the moment. A breaking change can't hurt in this case.
If you're working on that code and there is no major extra work for you, maybe you could [Spam bot protection #4](https://github.com/youngcut/kirby-form-blocks/issues/4) as well? Maybe you can benefit from it too.

**Comment by renestalder** [comment]
@youngcut What is the usage of the snippets/blocks in UIKit and starter-kit folders? Do I have to update their HTML too? If so, what are those snippets/blocks used for?
